### PR TITLE
Add an accessor for the fq_method

### PR DESCRIPTION
**Pull request by bdunne**

### CODE REVIEW

**Comment by Fryguy** [comment]
As discussed, I've avoided regexes in that split_raw_line method because of performance.  If we can find a way to not introduce *too* much performance overhead, then this would be good to me.

**Comment by Fryguy** [comment]
BTW, this will simplify things like https://github.com/ManageIQ/manageiq/blob/master/tools/log_processing/perf_timings.rb#L59

**Comment by bdunne** [comment]
I used a test evm.log file with 276472 lines.

Without start_with?:
```
real	0m1.144s
user	0m1.093s
sys	0m0.049s

real	0m1.142s
user	0m1.100s
sys	0m0.039s

real	0m1.176s
user	0m1.120s
sys	0m0.052s
```

With start_with?:
```
real	0m1.186s
user	0m1.133s
sys	0m0.048s

real	0m1.170s
user	0m1.124s
sys	0m0.041s

real	0m1.162s
user	0m1.110s
sys	0m0.048s
```

File: lib/gems/pending/util/miq_logger_processor.rb [hunk 1]
```diff
@@ -158,9 +158,10 @@ def self.split_raw_line(line)
     else
       q_task_id = nil
     end
+    fq_method = message[/^MIQ\(([\d\w\:\.\#\_\-]*)\)/, 1] if message.to_s.start_with?("MIQ(")
```
Comment (kbrock) [inline lib/gems/pending/util/miq_logger_processor.rb]: yea, remove the `starts_with?`

**Comment by bdunne** [comment]
It seems the `.to_s.` was slowing it down a bit.  @Fryguy suggested removing it and it appears to be a little bit faster.
```
        .to_s.      &.          &&          if message
real    0m1.808s    0m1.234s    0m1.226s    0m1.227s
real    0m1.231s    0m1.210s    0m1.199s    0m1.209s
real    0m1.201s    0m1.191s    0m1.176s    0m1.186s

user    0m1.285s    0m1.180s    0m1.173s    0m1.174s
user    0m1.183s    0m1.159s    0m1.147s    0m1.156s
user    0m1.156s    0m1.122s    0m1.108s    0m1.126s

sys     0m0.143s    0m0.065s    0m0.064s    0m0.057s
sys     0m0.044s    0m0.050s    0m0.049s    0m0.050s
sys     0m0.041s    0m0.047s    0m0.048s    0m0.049s
```

**Comment by kbrock** [comment]
nicely done guys

**Pull request merged**